src/api/routes/articleRouter.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import articleController from '../controllers/articleController';
+import checkArticle from '../../middlewares/getOneArticle';
+// import checkVerification from '../../middlewares/checkUserVerification';
+import checkArticleOwner from '../../middlewares/checkArticleOwnership';
+import checkValidToken from '../../middlewares/checkValidToken';
+
+const articleRouter = new Router();
+
+articleRouter.put('/:slug',
+  checkValidToken,
```

Comment:
before all checks, please, check if you are receiving a ```non-empty``` object

Reply:
yeah you are right